catalog/ui/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js

```diff
@@ -170,6 +176,17 @@ module.exports = function OpenlayersMap(insertionElement, selectionInterface, no
                     ], map)
                 });
             });
+
+            $(map.getTargetElement()).on('mouseleave', function() {
```

Comment:
The map can change positions when the mouse isn't on it (we fly to locations), we'll need to account for those scenarios as well.

Reply:
@andrewkfiedler What's the desired result when a user flies to a location? They click on a location on the side-pane and the coordinates update to that location?

Reply:
I think updating the location to be the map center would be fine (it'd theoretically be the location of the result).

Reply:
There are other scenarios though as well, such as using the gazetteer to fly to a location.

Reply:
Adjustments based on our conversation addressed in db5a713